testing/regress/ecl/translatedisk.ecl

```diff
@@ -17,15 +17,18 @@
 
 //version multiPart=false
 //version multiPart=true
+//version multiPart=true,optRemoteRead=true
```

Comment:
Should there be some variants that force the old activity ?

Reply:
imgkey.ecl will force the old activity.